packages/jest-stylelint-runner/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@commercetools-frontend/jest-stylelint-runner",
+  "version": "13.8.0",
+  "description": "Runner",
+  "bugs": "https://github.com/commercetools/merchant-center-application-kit/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/merchant-center-application-kit.git"
+  },
+  "keywords": [
+    "javascript",
+    "frontend",
+    "react",
+    "toolkit"
+  ],
+  "license": "MIT",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "src/index.js",
+  "module": "src/index.js",
```

Comment:
Given that we won't bundle the src files, can we move the fixtures/specs outside of `src` so that we don't ship them?